example-projects/axum-example/src/main.rs

```diff
@@ -23,16 +22,14 @@ fn echo_app() -> Router {
 #[tokio::main]
 async fn main() {
     if std::env::var("RUST_LOG").is_err() {
-        std::env::set_var("RUST_LOG", "axum_example=debug,tower_http=debug")
+        unsafe {
+            std::env::set_var("RUST_LOG", "axum_example=debug,tower_http=debug")
+        }
```

Comment:
Why is the `unsafe` needed here? Just curious.

Reply:
I vaguely remember it was needed because [set_var](https://doc.rust-lang.org/std/env/fn.set_var.html) is unsafe now. But I removed it and it seems fine. Not sure what happened before 🤷 ... I pushed a commit removing the two uses of unsafe.

Reply:
I forgot to sign off the commit, so I had to overwrite that commit history to fix it. 
Let me know if you'd like me to squash or clean things up in any other way.

Reply:
The pipeline still has a problem because of the signoff.